### DISCUSSION TITLE
I can't get code via email from Codemagic to login

### DISCUSSION THREAD

**Discussion by Marcelo-SmartEnvios** [codemagic-ci-cd/codemagic-docs#2454] [Q&A] [answered]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce
When i try to login to Codemagic i can't get any emails from Codemagic. I can't login and i can't to get a code via email.
https://prnt.sc/BF22Kg10-E6D

I see an discussion about this here: https://github.com/orgs/codemagic-ci-cd/discussions/1993 and i want help too

### Expected results

Get a code to login

### Actual results

No any emails.
https://prnt.sc/BF22Kg10-E6D

**Comment by dtrdic** [comment]
Hello @Marcelo-SmartEnvios please check your spam/junk folders as well. If there is a delay or if it's not arriving at all, it might be caused by spam/virus filters on your end. If you have implemented any spam filters, consider whitelisting our domain [no-reply@codemagic.io](mailto:no-reply@codemagic.io)

**Reply by Marcelo-SmartEnvios** [reply]
Hey @dtrdic ! Thanks for your response :smile: 
I'm still with this problem. I use an alias on gmail and i can filter emails that arrived by sender. I sent the verification code 3 times today, and my filter don't show any message on search. It seems to me that the email is not being sent.
If will help you, this is my email: `codemagic@smartenvios.com` and a print from my search on google admin console.

![image](https://github.com/codemagic-ci-cd/codemagic-docs/assets/91153932/3500d30a-f8ef-4868-8907-a4c8470563c1)

**Reply by mikhail-tokarev** [reply]
Hey @Marcelo-SmartEnvios 
here is the response from SMTP server
> Reason: smtp; 550-5.1.1 The email account that you tried to reach does not exist. Please try
550-5.1.1 double-checking the recipient's email address for typos

if you use alias make sure you gave permissions to receive emails from external addresses. I hope it helps

**Reply by Marcelo-SmartEnvios** [reply]
Hello @mikhail-tokarev ! Thanks for your response 😄
I don't understand this message you received, because if I send an email from my personal email (external) address it arrives normally, below is a print of a search for emails for the recipient which contains 2 test emails.
Is there any other way to help? Maybe change the email, if possible, or some other solution

![print console google](https://github.com/codemagic-ci-cd/codemagic-docs/assets/91153932/49bc796f-bfdc-495f-9aa6-5100ff5a413f)

**Reply by mikhail-tokarev** [reply]
I removed your address from the block list, can you give it another try?

**Reply by Marcelo-SmartEnvios** [reply, marked as answer]
It works!
Thank you so much!

**Reply by mikhail-tokarev** [reply]
nice to hear it's solved 🙌
